### PR TITLE
Fix/vet lint

### PR DESCRIPTION
**Pull request by Songmu**
pass `golint` and `go vet`

### CODE REVIEW

**Comment by hakobe** [comment]
:eyes:

**Comment by mattn** [comment]
hmmmm. squash please...

File: logging/logging.go [hunk 1]
```diff
@@ -33,55 +34,63 @@ func stringToLogLevel(name string) *LogLevel {
...
 
-func ConfigureLoggers(rootLogLevel string) {
-	logLevelConfigs["root"] = stringToLogLevel(rootLogLevel)
+// ConfigureLoggers XXX
+func ConfigureLoggers(rootlogLevel string) {
+	logLevelConfigs["root"] = stringTologLevel(rootlogLevel)
 }
 
-func (logger *Logger) currentLogLevel() *LogLevel {
+func (logger *Logger) currentlogLevel() *logLevel {
```
Comment (hakobe) [inline logging/logging.go]: [nit]
this shoud be `currentLogLevel`

**Comment by hakobe** [comment]
LGTM with comments.

**Comment by Songmu** [comment]
squash merged.

**Comment by mattn** [comment]
LGTM

**Pull request closed without merging**